nekoyume/Assets/_Scripts/UI/Widget.cs

```diff
@@ -81,6 +82,18 @@ protected virtual void Awake()
 
             CloseWidget = () => Close();
             SubmitWidget = null;
+
+            AnimationState.Subscribe(stateType =>
+            {
+                var fields = GetType().GetFields(System.Reflection.BindingFlags.NonPublic |
+                                                 System.Reflection.BindingFlags.Instance);
```

Comment:
Can remove the `NonPublic` binding flag? It can mistake when `Selectable` component is `public`.